core/templates/dev/head/editor/state_editor_responses.html

```diff
@@ -77,7 +77,11 @@
                                        on-save-answer-group-feedback="saveDefaultOutcomeFeedback"
                                        on-save-answer-group-dest="saveDefaultOutcomeDest"
                                        is-editable="editabilityService.isEditable()"
+<<<<<<< HEAD
```

Comment:
Also, just to check, did you test this manually? This should probably have shown up in the UI...

In any case, just wanted to suggest making sure, via manual testing, that the changes in here definitely do the right thing under different circumstances, especially since this PR doesn't come with automated tests.